tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -158,6 +177,18 @@ pub struct CompressedCiphertext {
     pub pbs_order: PBSOrder,
 }
 
+impl ParameterSetConformant for CompressedCiphertext {
+    type ParameterSet = CiphertextConformanceParams;
+
+    fn is_conformant(&self, param: &CiphertextConformanceParams) -> bool {
+        self.ct.is_conformant(&param.ct_params)
+            && param.message_modulus == self.message_modulus
+            && param.carry_modulus == self.carry_modulus
+            && param.pbs_order == self.pbs_order
+            && self.degree.0 == self.message_modulus.0 - 1
```

Comment:
use param on the right, should limit mistakes

Reply:
I can do it, but I don't understand what it changes in preventing mistakes

Reply:
if someone forgets to check the param.message_modulus == self.message_modulus we are comparing only with data in the ciphertext, but the proposal to create a ConformanceParam struct may allow to make this more easily